### PR TITLE
[WIP] 1.10

### PR DESCRIPTION
**Pull request by Filipvds**

### CODE REVIEW

**Comment by achim-k** [comment]
Still a long way to go.

The navigation menu has to many entries. It always jumps when I click on one of them. We should reduce the number of entries significantly.

File: docs/quick-start/test-the-setup.rst [hunk 1]
```diff
@@ -21,6 +21,6 @@ processor can be checked:
 
 .. image:: /assets/images/First-steps/Network-settings.png
 
-.. tip:: In case of failure, check if the cables are properly connected and
+.. note:: In case of failure, check if the cables are properly connected and
```
Comment (achim-k) [inline docs/quick-start/test-the-setup.rst]: I think you should start a new line after `note::`

**Comment by adolfo-rt** [comment]
Is this PR still relevant or can it be closed?

**Comment by achim-k** [comment]
Can be closed.

**Pull request closed without merging**